### DISCUSSION TITLE
활성화 스쿼드 아이디어 살 붙이기

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#909] [Ideas]
# 할 일
- 이니셔티브 후보 주제에 자신의 생각와 시나리오 등 기능을 상세할 수 있는 아이디어를 코멘트로 남겨 주세요.
- **7월 6일 오전 9시까지 모든 멤버가 코멘트 남기기**
- 7월 6일 회의에서는 코멘트 달린 내용으로 의견 합치기
  - 후이가 리딩해서 회의 시간 픽스 및 진행

# 이니셔티브 후보
- 다른 서비스 컨텐츠 가져오기 기능
- studylog에 질답을 등록하는 템플릿 기능
- 댓글 기능
- 추천 게시글

**Comment by kwannee** [comment]
### 추천 게시글

학습 로그 제일 하단에 현재 읽고 있는 게시글과 비슷한 주제의 `추천 게시글` 목록이 나온다. 

학습 로그를 작성할 때 함께 명시하는 `tags`를 기반으로 게시글을 선정한다.

선정 기준으로는 다음과 같은 요인이 있을 수 있다.

1. 조회수, 추천 수, 스크랩 수
2. 겹치는 태그가 많은 순
3. 작성일이 최근인 순

1번은 기존 인기있는 학습 로그를 선정하는 기준을 재활용할 수 있을 것 같다.

2번은 해당 내용과 가장 겹치는 내용을 찾을 수 있을 것 같다.

3번은 해당 태그와 관련해서 가장 최근인 만큼 비슷한 이슈를 다루고 있을 가능성이 높을 것 같다.

**Comment by Yboyu0u** [comment]
- 다른 서비스 컨텐츠 가져오기 기능
노션에 있는 enmbed, bookmark 기능추가해서 다른컨텐츠에 대한 가독성을 높이는 방법
![image](https://user-images.githubusercontent.com/63579113/177309283-65501d02-9d66-4e5a-ae5a-78b118811d56.png)
 
- studylog에 질답을 등록하는 템플릿 기능
댓글 기능과 비슷한 것 같은데 댓글로 일단은 해결할 수 있을 것 같다. 

- 댓글 기능
우선은 깊에 생각하지 않고 답글 다는 기능과 그 답글에 답글다는 기능이 추가되는 쪽으로 생각하는 게 나을 것 같다.

- 추천 게시글
자신이 많이 찾아본 글이나 자신이 작성한 글에서 연관성을 찾아 게시글을 추천해 준다. -> 해시태그를  기준으로

**Comment by her0807** [comment]
### studylog에 질답을 등록하는 템플릿 기능
아래와 같이 공부한 내용은 URL 로 걸어두고, 잉의 의견처럼 노션에 있는 enmbed, bookmark 기능추가해서 보여주면 좋을 듯하고
그 내용을 공부했을 때 고민했던 부분을 본인이 직접 Q 를 적고, 해당 부분에 대해 내가 내린 결론을 A 에 적는 식으로 
하면 복습도 되고, 다른 사람이 겪었던 트러블 슈팅 공유도 활성화되고 좋을 듯 하다. 

<img width="790" alt="스크린샷 2022-07-05 오후 7 56 20" src="https://user-images.githubusercontent.com/26570275/177312753-24518ec7-fab5-441f-b97b-884dbe6c7dcc.png">

**Comment by wishoon** [comment]
## 학습로그 댓글 기능

- 학습로그 하단에 해당 글에 대해서 글을 통해 본인의 생각을 작성할 수 있는 기능을 제공한다. (= 댓글)
- 해당 댓글에도 공감 기능을 넣어 best 댓글 기능을 제공해도 괜찮을 것으로 보임
- 댓글에 대한 대댓글을 통해서 글을 통해 토론을 할 수 있도록 기능을 제공해도 괜찮을 것으로 보임

## 학습로그 글 추천기능

- 학습로그 하단에 해당 글과 관련된 내용을 추천해주는 기능을 제공한다. (ex. velog)
- 학습 로그를 작성할 때 함께 올릴 수 있는 tags를 이용하여 게시글을 선정한다.
    - 예상되는 문제점
        - 현재 tags들이 `#` 없이 작성되어 있는 경우도 있음
        - tags들이 유의미하지 않은 이름으로 등록된 경우도 있음 (ex. #루키, #에덴)

**Comment by euijinkk** [comment]
## 소통 강화
- 게시물 좋아요 쉽게 누를 수 있도록 하기
- 댓글 기능 구현 / 대댓글 기능 구현
- 댓글 좋아요 기능 구현
- 자신의 게시물에 댓글이 달렸을 때 이메일 가도록 구현

- 인기있는 학습로그 10개라서 의미가 퇴색되는 것 같다.
- 각주마다, 프론트엔드 백엔드 1/2명만 뽑아보는건 어떨까? 훨씬 스포트라이트 되어서 재밌을 것 같다!

## 필터링 기능 강화
- 프론트엔드/백엔드 필터링이 어려움.
- 강의별 필터링이 중요할까?

**Comment by verus-j** [comment]
# 다른 서비스 컨텐츠 가져오기 및 추천 게시글
* 벨로그나 티스토리에서 추천하는 블로그 글을 간단하게 리스트로 보여주기
* 크루원들이 작성한 블로그 글을 프롤로그로 가져와서 보여주기
* 관심있는 주제의 태그를 선택해서 관련된 주제의 외부의 글을 추천 리스트로 보여주기

# studylog에 질답을 등록하는 템플릿 및 댓글 기능
* 작성한 학습로그에서 작성자가 토론하기 원하는 주제의 질문을 학습로그와 함께 작성하고 크루원들이 들어와서 학습로그를 읽고 각자의 생각을 답으로 적고 토론하기

**Comment by al-bur** [comment]
## 게시물 가져오기 + 질의응답 기능

- 기본적으로, 게시물을 가져올 수 있는 기능
  - 기존에 존재하고 있는 블로그와 정면 승부로는 힘들다.
  - 편하게, 여러 블로그들을 가져와서 기록해놓는 용으로 사용해도 괜찮을듯?
- 가져온 게시물에 대해 스스로 돌아볼 수 있도록 Q&A 달 수 있는 템플릿 제공
  - 본인이 작성한 Q&A에 다른 크루들이 의견을 달 수 있도록 댓글 기능과 연계하면 풍부한 이야기가 오고갈 수 있을 것 같다.

## 추천 게시글 기능

- 기본적으로 글들이 가지고 있는 `태그`를 바탕으로 추천해줘도 될 것 같다.
